statsmodels/compat/python.py

```diff
@@ -232,3 +232,16 @@ def get_class(func):
     except AttributeError:
         #Python 3
         return func.__self__.__class__
+
+
+def with_metaclass(meta, *bases):
+    """Create a base class with a metaclass."""
+    # This requires a bit of explanation: the basic idea is to make a dummy
+    # metaclass for one level of class instantiation that replaces itself with
+    # the actual metaclass.
+    class metaclass(meta):
+        def __new__(cls, name, this_bases, d):
+            return meta(name, bases, d)
+    return type.__new__(metaclass, 'temporary_class', (), {})
+
+from itertools import combinations
```

Comment:
any reason not to put this at the top of the file with other imports?

Reply:
Based on a file search, it looks like this is not used anymore.
I delete it and see what appveyor and travis say.

Reply:
I've got a PR that explicitly goes through compat and identifies unneeded stuff